src/Microsoft.AspNet.Mvc.Core/ModelBinding/SimpleTypeModelBinder.cs

```diff
@@ -8,12 +8,10 @@
 
 namespace Microsoft.AspNet.Mvc.ModelBinding
 {
-    public sealed class TypeConverterModelBinder : IModelBinder
+    public class SimpleTypeModelBinder : IModelBinder
     {
         public async Task<ModelBindingResult> BindModelAsync(ModelBindingContext bindingContext)
         {
-            ModelBindingHelper.ValidateBindingContext(bindingContext);
```

Comment:
Not sure removing `ValidateBindingContext()` fits in this PR at all. Let's discuss this separately.